src/store/reducers/init-game.js

```diff
@@ -4,7 +4,7 @@ const initialState = {
     correctWords: [],
 };
 
-const reducer = (state = initialState, { type, correctWords }) => {
+const initGameReducer = (state = initialState, { type, correctWords }) => {
```

Comment:
and reducers correspond to state so their names are in the form of `somethingReducer`